catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/EventProcessorImpl.java

```diff
@@ -74,7 +77,12 @@
 
     private Map<String, ServiceRegistration> existingSubscriptions;
 
-    private final ExecutorService threadPool = Executors.newCachedThreadPool();
+    private final ExecutorService threadPool = new ThreadPoolExecutor(0,
```

Comment:
❓ Is there a reason you changed this to manual construction of your threadpool instead of just calling `Executors.newCachedThreadPool(StandardThreadFactoryBuilder.newThreadFactory("eventProcessorThread"))`

Reply:
I *think* I was looking at the use of some of these threads and seeing if a different executor would provide a more optimized behavior. It seems that it ended up back where it started with the same call that a newCachedThreadPool would. Making the call you reference would provide the same effect in a more concise way. :+1: